Okta.Xamarin/Okta.Xamarin.Test/OktaContextShould.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Okta.Xamarin.Test
+{
+	public class OktaContextShould
+	{
+		[Fact]
+		public void RaiseSignInEventsOnSignIn()
+		{
+			OidcClient client = new OidcClient(new OktaConfig("testoktaid", "https://dev-00000.oktapreview.com", "com.test:/redirect", "com.test:/logout"));
+			OktaContext.Init(client);
+			bool? signInStartedEventRaised = false;
+			bool? signInCompletedEventRaised = false;
+			OktaContext.AddSignInStartedListener((sender, eventArgs) => signInStartedEventRaised = true);
+			OktaContext.AddSignInCompletedListener((sender, eventArgs) => signInCompletedEventRaised = true);
+			
+			OktaContext.Current.SignIn();
```

Comment:
Probably to handle in a separate PR, but it seems we use both `Sign In` and `Login` for the same action. I'd be great we use one of those for consistency.

Reply:
Agreed.